ReactAndroid/src/main/java/com/facebook/systrace/SystraceMessage.java

```diff
@@ -7,17 +7,19 @@
 
 package com.facebook.systrace;
 
-/** Systrace stub. */
+import java.util.ArrayList;
+import java.util.List;
+
 public final class SystraceMessage {
 
-  private static final Builder NOOP_BUILDER = new NoopBuilder();
+  public static Boolean INCLUDE_ARGS = false;
```

Comment:
Adding arguments to the section name makes grouping similar events a lot harder so by default I disable it. I don't really see how else args are useful for systrace in Android Studio. How is that used internally at facebook?